Chap_API_Data_Mgmt.tex

```diff
@@ -337,7 +339,7 @@ \subsection{\code{PMIx_Data_print}}
 }
 
 \begin{arglist}
-\argin{output}{The address of a pointer into which the address of the resulting output is to be stored. (\code{char**})}
+\argout{output}{The address of a pointer into which the address of the resulting output is to be stored. (\code{char**})}
 \argin{prefix}{String to be prepended to the resulting output (\code{const char*})}
```

Comment:
double check similar API's to see if they use argin or argout for this

Reply:
This change looks good to me after review.  Consistent with other uses of argout